ctms/schemas/contact.py

```diff
@@ -127,6 +128,21 @@ class CTMSResponse(BaseModel):
     vpn_waitlist: VpnWaitlistSchema
 
 
+class CTMSOptionalResponse(BaseModel):
```

Comment:
I think the docstring needs updating, looks like it was copied.

I'm OK to use this, but we'll need to check with Cinchy if it is OK if the responses are different from ``/ctms`` and ``/ctms/{email_id}``. Here's what a record with missing group data looks like:

```json
{
  "items": [
    {
      "amo": null,
      "email": {
        "primary_email": "jwhitlock@mozilla.com",
        "basket_token": "8b359504-d1b4-4691-9175-cfee3059b171",
        "double_opt_in": false,
        "sfdc_id": null,
        "first_name": null,
        "last_name": null,
        "mailing_country": "us",
        "email_format": "T",
        "email_lang": "es",
        "has_opted_out_of_email": false,
        "unsubscribe_reason": null,
        "email_id": "e1d35779-9f14-4553-b2aa-85f9629f68bb",
        "create_timestamp": "2021-04-07T16:56:36.579296+00:00",
        "update_timestamp": "2021-04-07T16:56:36.579296+00:00"
      },
      "fxa": null,
      "mofo": null,
      "newsletters": [],
      "vpn_waitlist": null
    }
]
}
```

With the other types, the ``"fxa": null,`` and similar items would instead be the default values.

Reply:
Ah, I wasn't 100% sure what should happen in this case. I think maybe the removal of the optional class could align